### PR TITLE
Assignment-1 initial commit

### PR DESCRIPTION
**Pull request by pranav96vaidya**
@anjanaNewput please review it.

### CODE REVIEW

File: Assignment 1/index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                        <td>Moscow</td>
+                        <td>Gosudarstvenny gimn Rossiyskoy Federatsii</td>
+                    </tr>
+                </tbody>
+                <tfoot></tfoot>
+            </table>
+
+            <div>
+                <form action="#" method="post">
+                    <label name="Name"><b>Name:</b></label>
```
Comment (anjanaNewput) [inline Assignment 1/index.html]: correct it, name should be in lower case

File: Assignment 1/index.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    <input type="text" maxlength="30" required>
+                    <br><br>
+                    <label name="Gender"><b>Gender</b></label><br><br>
```
Comment (anjanaNewput) [inline Assignment 1/index.html]: same here

File: Assignment 1/theWorld.html [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+<body>
+    <header>
+        <h1>The World</h1>
+    </header>
+
+    <section style="width:100%;height:70%">
+
+        <nav>
+            <ul>
+                <li><a href="#India">India</a></li>
```
Comment (anjanaNewput) [inline Assignment 1/theWorld.html]: id and class name should be in lower case file name should be in camelcase

File: Assignment 1/index.html (hunk 1 above)
Comment (pranav96vaidya) [reply Assignment 1/index.html]: Done.

**Pull request merged**